.changeset/tasty-pigs-tickle.md

```diff
@@ -0,0 +1,12 @@
+---
+"@spectrum-css/preview": major
```

Comment:
@castastrophe- with the config updates, additions to the env file, the dependency technically changed, I think major seems appropriate.